tests/integration_tests/test_subtensor.py

```diff
@@ -75,19 +75,15 @@ def test_network_overrides( self ):
         assert subtensor.endpoint_for_network() == bittensor.__nobunaga_entrypoint__
 
     def test_connect_no_failure( self ):
-        self.subtensor.connect(timeout = 1, failure=False)
-
+        self.subtensor.substrate
+        assert True
```

Comment:
What is this testing?

sounds like just no unexpected errors, is this test for that?